gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -1127,34 +1134,27 @@ public Long megabytesCopiedPerChunk() {
 
     /**
      * Creates a copy request. {@code target} parameter is used to override source blob information
-     * (e.g. {@code contentType}, {@code contentLanguage}). {@code target.contentType} is a required
-     * field.
+     * (e.g. {@code contentType}, {@code contentLanguage}).
      *
      * @param sourceBucket name of the bucket containing the source blob
      * @param sourceBlob name of the source blob
      * @param target a {@code BlobInfo} object for the target blob
      * @return a copy request
-     * @throws IllegalArgumentException if {@code target.contentType} is {@code null}
      */
-    public static CopyRequest of(String sourceBucket, String sourceBlob, BlobInfo target)
-        throws IllegalArgumentException {
-      checkContentType(target);
+    public static CopyRequest of(String sourceBucket, String sourceBlob, BlobInfo target) {
```

Comment:
Fine with you prior change of calling target targetId or targetInfo though my preference would have been to leave it target in all places as before.